### ISSUE TITLE
The GUI doesn't work

### ISSUE THREAD

**Issue by emyabdel** [ahmadudin/electron-instaPy-GUI#52]
hello

The GUI doesn't work to run instapy, there are many issues every time I resolve an issue another one comes. could you please tell the developer to make an update or something else?

**Comment by jcpsimmons** [comment]
Agreed, please mark as not working.

**Comment by ahmadudin** [comment]
Yeah, this GUI is so outdated compared to instaPy. I'll update it, maybe in the next 2 month.

**Comment by Rubenzella** [comment]
Hi ahmadudin, any news in regards of the updated GUI?

**Comment by ahmadudin** [comment]
Sorry @Rubenzella , i afraid that i don't have the time to do the update yet.
